astilectron-bundler/main.go

```diff
@@ -23,6 +25,10 @@ var (
 	windows           = flag.Bool("w", false, "if set, will add windows/amd64 to the environments")
 )
 
+func init() {
+	flag.Var(ldflags, "ldflags", "extra values to concatenate onto -ldflags, eg X:main.Version=1.0.7")
```

Comment:
What is the gain of putting this in the `init` function?

I would have put it just above, with the rest of the flag declarations

Reply:
flag.Var takes a reference and as far as I know there's not a version that returns a pointer like the other flag methods you're using, so I don't see a way to do what you are asking.  Am I missing something?